src/test/ui/assume-type-intrinsics.rs

```diff
@@ -0,0 +1,13 @@
+// error-pattern: any use of this value will cause an error
```

Comment:
Oh, I only just realized this file is in the wrong folder. It should be in an appropriate subfolder, in this case `consts` sounds best. I'll take care of that as part of https://github.com/rust-lang/rust/pull/79910.